src/options/swap_options.py

```diff
@@ -11,6 +11,7 @@ def initialize(self):
 		self.parser.add_argument('--num_seg_cls', type=int, default=12,help='Segmentation mask class number')
 		self.parser.add_argument('--train_G', default=True, type=bool, help='Whether to train the model')
 		self.parser.add_argument('--device', default='cuda:0', type=str, help='Which GPU(s) to use')
+		self.parser.add_argument('--lap_bld', action='store_true', help='Whether to use Laplacian multi-band blending')
```

Comment:
How bout setting this option to True by default?

Reply:
Now the blending result is not good, it is better to inpainting the mask mismatch area (represented by hole_map in the code), and then use the blending

Reply:
Add this option to README